pkgs/googleapis_firestore_v1/lib/google/api/annotations.pb.dart

```diff
@@ -0,0 +1,29 @@
+//
+//  Generated code. Do not modify.
```

Comment:
Consider adding a .gitattributes file to mark `.pb.dart` and `.pbenum.dart` files as `linguist-gernated=true`.

https://github.com/dart-lang/http/blob/master/.gitattributes

Reply:
done!